pkg/extensions/search/README.md

```diff
@@ -0,0 +1,513 @@
+          "Digest": "sha256:650d596072ad45c6b74f4923e2cfea8158da2fb3a7b8dbb0b9ae4da3088d0591"
+        }
+      }
+    }
+  }
+}
+```
+
+# Global search
+
+Sample request
+```
+{
+  GlobalSearch(query: "ubuntu:latest") {
+    Images {
+      RepoName
+      Tag
+      LastUpdated
+      Layers {
+        Size
+        Digest
+      }
+    }
+  }
+}
+```
+
+**Sample response**
+```
+{
+  "data": {
+    "GlobalSearch": {
+      "Images": [
+        {
+          "RepoName": "ubuntu",
+          "Tag": "latest",
+          "LastUpdated": "2022-10-25T01:53:41.769246372Z",
+          "Layers": [
+            {
+              "Size": "30426374",
+              "Digest": "sha256:301a8b74f71f85f3a31e9c7e7fedd5b001ead5bcf895bc2911c1d260e06bd987"
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+```
+Sample request
+```
+{
+  GlobalSearch(query: "") {
+    Repos {
+      Name
+    }
+  }
+}
+```
+**Sample response**
+```
+{
+  "data": {
+    "GlobalSearch": {
+      "Repos": [
+        {
+          "Name": "centos"
+        },
+        {
+          "Name": "ubuntu"
+        }
+      ]
+    }
+  }
+}
+```
+# Search derived images
+Sample query
+```
+{
+  DerivedImageList (image: "ubuntu") {
+    RepoName
+    Tag
+    LastUpdated
+  }
+}
+```
+**Sample response**
+```
+{
+  "data": {
+    "DerivedImageList": [
+      {
+        "RepoName": "mariadb",
+        "Tag": "latest",
+        "LastUpdated": "2022-10-18T14:56:33.1993083+03:00"
+      },
+      {
+        "RepoName": "maven",
+        "Tag": "latest",
+        "LastUpdated": "2022-10-14T18:30:12.0929807+03:00"
+      },
+      {
+        "RepoName": "tomcat",
+        "Tag": "jre17",
+        "LastUpdated": "2022-10-18T14:55:13.8303866+03:00"
+      }
+    ]
+  }
+}
+```
+
+# Search base images
+Sample query
+```
+{
+  BaseImageList (image: "mariadb") {
+    RepoName
+    Tag
+    LastUpdated
+  }
+}
+```
+
+**Sample response**
+```
+{
+  "data": {
+    "BaseImageList": [
+      {
+        "RepoName": "mariadb",
+        "Tag": "latest",
+        "LastUpdated": "2022-10-18T14:56:33.1993083+03:00"
+      },
+      {
+        "RepoName": "ubuntu",
+        "Tag": "jammy",
+        "LastUpdated": "2022-10-14T18:29:18.0325322+03:00"
+      },
+      {
+        "RepoName": "ubuntu",
+        "Tag": "latest",
+        "LastUpdated": "2022-10-14T18:26:59.6707939+03:00"
+      }
+    ]
+  }
+}
+```
+
+# Search image
+Sample query
+```
+{ 
+  Image(image: "mariadb:latest") { 
+    RepoName
+    Tag
+    LastUpdated
+    Digest
+    Description
+  }
+}
+```
+
+**Sample response**
+```
+{
+  "data": {
+    "Image": {
+      "RepoName": "mariadb",
+      "Tag": "latest",
+      "LastUpdated": "2022-10-18T14:56:33.1993083+03:00",
+      "Digest": "sha256:49a299f5c4b1af5bc2aa6cf8e50ab5bad85db4d0095745369acfc1934ece99d0",
+      "Description": "MariaDB Server is a high performing open source relational database, forked from MySQL."
+    }
+  }
+}
+```
+
```

Comment:
I think it is still useful to have an example using `curl` (with a complete url with the graphql endpoint, query parameters, headers), just like in the old document. Maybe add a new section at the beginning of the doc for this.

Reply:
Added an example with curl below the query table